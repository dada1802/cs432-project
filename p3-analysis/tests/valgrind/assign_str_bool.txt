==695467== Memcheck, a memory error detector
==695467== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==695467== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==695467== Command: ../decaf inputs/assign_str_bool.decaf
==695467== 
Type mismatch: bool is incompatible with str on line 9
==695467== 
==695467== HEAP SUMMARY:
==695467==     in use at exit: 0 bytes in 0 blocks
==695467==   total heap usage: 2,007 allocs, 2,007 frees, 245,348 bytes allocated
==695467== 
==695467== All heap blocks were freed -- no leaks are possible
==695467== 
==695467== For lists of detected and suppressed errors, rerun with: -s
==695467== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
