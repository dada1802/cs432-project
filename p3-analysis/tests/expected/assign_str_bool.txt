Type mismatch: bool is incompatible with str on line 9
